README.md

```diff
@@ -245,6 +247,12 @@ If your desired ghc has been found, you use it to install haskell-language-serve
 ./cabal-hls-install hls-8.6.5
 ```
 
+### Installation from Hackage
+
+Direct installation from Hackage is supported via `cabal install haskell-language-server`.
+This builds the `haskell-language-server` binary and installs it in the default Cabal binaries folder.
+This binary will only work against the same GHC version that was used to build it.
```

Comment:
Maybe include the words "but not recommended"?

Reply:
```suggestion
This binary will only work with projects built using the same GHC version that was used to build the binary itself.
```

Reply:
> 
> 
> Maybe include the words "but not recommended"?

Or something like "So it is recommended for advanced users."?